auth-api/src/auth_api/services/user.py

```diff
@@ -190,10 +194,33 @@ def delete_otp_for_user(user_name, token_info: Dict = None):
         check_auth(org_id=org_id, token_info=token_info, one_of_roles=(ADMIN, COORDINATOR, STAFF))
         try:
             KeycloakService.reset_otp(str(user.keycloak_guid))
+            User.send_otp_authenticator_reset_notification(user.email, origin_url)
         except HTTPError as err:
             current_app.logger.error('update_user in keycloak failed {}', err)
             raise BusinessException(Error.UNDEFINED_ERROR, err)
 
+    @staticmethod
+    def send_otp_authenticator_reset_notification(receipt_email, origin_url):
```

Comment:
typo receipt_email ;recipient email i guess

Reply:
oops.. yes.. will correct it